integration/spark-common/src/main/scala/org/apache/carbondata/streaming/StreamSinkFactory.scala

```diff
@@ -255,6 +256,29 @@ object StreamSinkFactory {
       optionsFinal.put("fileheader", carbonTable.getCreateOrderColumn(carbonTable.getTableName)
         .asScala.map(_.getColName).mkString(","))
     }
+    val badRecordsFromLoad = parameters.get("bad_record_path")
+    val badRecordsFromCreate = carbonTable.getTableInfo.getFactTable.getTableProperties
+      .get("bad_records_path")
+    val badRecordsPath = if (badRecordsFromLoad.isDefined &&
```

Comment:
badRecordsFromLoad.isDefined  check not required

Reply:
badRecordsFromLoad if Option[String] to get the value from it i need to check isDefined and then get the value.